### PR TITLE
Updating styles for the reader page and font family

### PR DESCRIPTION
**Pull request by mateus**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <head>
     <title><%= content_for?(:title) ? content_for(:title) : "Contentify" %></title>
     <%= csrf_meta_tags %>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (Goldenson) [inline app/views/layouts/application.html.erb]: 📱

Comment (mateus) [inline app/views/layouts/application.html.erb]: 📱

File: app/assets/stylesheets/basis/_base.scss [hunk 2]
```diff
@@ -3,32 +3,103 @@
 }
 
 body {
-  font-size: 24px;
-  font-family: "Sentinel", "helvetica";
+  font-size: 23px;
+  font-family: "Tiempos", "helvetica";
```
Comment (mateus) [inline app/assets/stylesheets/basis/_base.scss]: @kaigenji I'm removing Sentinel and adding Tiempos here.

Comment (kaigenji) [inline app/assets/stylesheets/basis/_base.scss]: 23 pixels haha?

Comment (mateus) [inline app/assets/stylesheets/basis/_base.scss]: I'll show you how it's like on heroku. 22px seemed too little, 24px too much ahha. We can change that in another PR later too.

**Comment by kaigenji** [comment]
:shipit:

**Pull request merged**